UnitTests/Tests/WeatherUpdateSpec.swift

```diff
@@ -0,0 +1,61 @@
+@testable import Tropos
+import CoreLocation
+import Quick
+import Nimble
+
+private func weatherConditions(temperature temperature: Int = 90, precipitationProbability: String? = nil, precipitationType: String? = "rain") -> [String: AnyObject] {
+    var data: [String: AnyObject] = [
+        "temperatureMin": 50,
+        "temperatureMax": 60,
+    ]
+
+    if let probability = precipitationProbability {
+        data["precipProbability"] = probability
+        if let type = precipitationType {
+            data["precipType"] = type
+        }
+    }
+
+    return [
+        "currently": ["temperature": temperature],
+        "daily": ["data": [data]],
+    ]
+}
+
+final class WeatherUpdateSpec: QuickSpec {
+    override func spec() {
+        describe("TRWeatherUpdate") {
+            context("currentTemp is higher than currentHigh") {
+                it("updates currentHigh to match") {
+                    let conditions = weatherConditions(temperature: 70)
```

Comment:
I think I must have an internal line number threshold for determining whether to insert extra line breaks in these situations. It appears to be at around 4 lines. 😛